.classpath

```diff
@@ -10,6 +10,5 @@
 	<classpathentry kind="var" path="USERLIBS_DIR/CTRE_Phoenix.jar"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/CTRE_Phoenix-sources.jar"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/navx_frc.jar"/>
-	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
```

Comment:
This should not be deleted. Revert/Stash this file (so it's not changed at all from master) and make sure it's included in Eclipse (in the Java Build Path)